### PR TITLE
fix #987:Enhancing the document upload and identifier dialog

### LINKED ISSUE #987: Enhancing the Dialog fragments
**Summary:** 

Currently, the Dialog fragments such as Document dialog, identifier dialog lacks UI polish. This task would comprise of enhancing the UI of the dialogs fragments

**Screenshots:** 

**DocumentUploadDialog**
screenshot_2018-05-25-12-41-12

**Identifier**
screenshot_2018-05-25-23-44-45

### PR DESCRIPTION
**Pull request by AkshGautam**
fix #987 
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by AkshGautam** [comment]
@therajanmaurya @tarun0  please review
screenshot_2018-05-25-12-45-35

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/documentdialog/DocumentDialogFragment.java [hunk 1]
```diff
@@ -67,6 +67,9 @@
     @BindView(R.id.tv_choose_file)
     TextView tv_choose_file;
 
+    @BindView(R.id.btn_browse_document)
+    Button bt_browse;
```
Comment (therajanmaurya) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/documentdialog/DocumentDialogFragment.java]: Please follow the camel case naming convention

Comment (AkshGautam) [reply mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/documentdialog/DocumentDialogFragment.java]: @therajanmaurya snake case convention is followed across the entire document. For example : tv_choose_file,et_document_description

Comment (AkshGautam) [reply mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/documentdialog/DocumentDialogFragment.java]: @therajanmaurya done

**Pull request merged**